src/common/converter.ts

```diff
@@ -54,7 +54,7 @@ export function devWorkspaceToDevfile(devworkspace: IDevWorkspace): IDevWorkspac
         template.projects = devworkspace.spec.template.projects;
     }
     if (devworkspace.spec.template.components) {
-        template.components = devworkspace.spec.template.components;
+        template.components = filterPluginComponents(devworkspace.spec.template.components);
```

Comment:
we should not hardcode `schemaVersion: '2.0.0'` instead we should store that into devworkspace annotations.
It's not really related to this issue and I'm not sure we have a dedicated issue for that, but we should. Maybe it can be part of https://github.com/eclipse/che/issues/19430

Reply:
I was just going to do it as part of https://github.com/eclipse/che/issues/19430 just so everything stored in annotations is done at once